control/timeresp.py

```diff
@@ -249,7 +249,7 @@ def forced_response(sys, T=None, U=0., X0=0., transpose=False,
     Notes
     -----
     For discrete time systems, the input/output response is computed using the
-    :scipy-signal:ref:`scipy.signal.dlsim` function.
+    scipy.signal.dlsim function.
```

Comment:
Just fix the broken intersphinx mapping (untested):

```suggestion
    :scipy:ref:`scipy.signal.dlsim` function.
```

https://github.com/python-control/python-control/blob/ec8cbf053a8dee367e925f54eb42aa256a8d3265/doc/conf.py#L99-L101

Reply:
Thanks! I've updated the commits to fix it.